vision_datasets/common/data_manifest.py

```diff
@@ -848,10 +855,18 @@ def process_labels_without_categories(image):
             images = [x for x in images_by_id.values()]
             return DatasetManifest(images, None, data_type)
 
-        cate_id_name = [(cate['id'], cate['name']) for cate in coco_manifest['categories']]
+        supercategory_field_in_categories = False
```

Comment:
maybe it's worth to make it a dataset property (self.supercategory_field_in_categories), so everywhere else (including the evaluation code), you can directly access this property